### PR TITLE
fix issue with Monad.do #4

### PR DESCRIPTION
**Pull request by safareli**
fix issue with Monad.do #4

### CODE REVIEW

File: free.js [hunk 1]
```diff
@@ -34,13 +34,22 @@ Free.prototype.chain = function(f) {
 
 const liftF = command => Free.Impure(command, Free.Pure)
 
+const IGNORE_VALUE = typeof Symbol != "undefined"
+  ? Symbol.for("free/ignore_value")
+  : "@@free/ignore_value"
+
```
Comment (davidchambers) [inline free.js:39]: Will symbols cause us some of the same headaches as [`instanceof` checks](https://github.com/sanctuary-js/sanctuary/issues/100)?

Comment (safareli) [inline free.js:39]: I dont know if that will have some bad effects as it's internal thing. but your question made me to think on much better solution which I have just pushed.
And if we move Monad's do function into Free.do we would not need to export this too.

**Comment by safareli** [comment]
@DrBoolean can you take a look at it :3 ?

**Comment by DrBoolean** [comment]
My goodness, I didn't see until just now! Man, i'm bad at maintaining this project. I still have to check the other huge PR/rewrite.

Let's do this for sure

**Pull request merged**